### ISSUE TITLE
Warning in MarkupAdminDataTable.module

### ISSUE THREAD

**Issue by matjazpotocnik** [processwire/processwire-issues#1094]
Looking at the comments on row() method:

```
/**
 * Add a row to the table
 * 
 * @param array $a Array of columns that will each be a `<td>`, where each element may be one of the following:
 *   - `string`: converts to `<td>string</td>`
 *   - `array('label' => 'url')`: converts to `<td><a href='url'>label</a></td>`
 *   - `array('label', 'class')`: converts to `<td class='class'>label</td>`
 * @param array $options Optionally specify any one of the following:
 *   - separator (bool): specify true to show a stronger visual separator above the column
 *   - class (string): specify one or more class names to apply to the `<tr>`
 *   - attrs (array): array of attr => value for attributes to add to the `<tr>`
 * @return $this
 * 
 */
```

If I use this:

```
$table->row(array(
  array('foo' => 'bar'),
));
```

then the warning is issued:

```
PHP Notice: Undefined offset: 1 in ...\MarkupAdminDataTable.module:289
PHP Notice: Undefined offset: 0 in ...\MarkupAdminDataTable.module:289
```

Probably the same as https://github.com/processwire/processwire-issues/issues/1084

**Comment by ryancramerdesign** [comment]
Thanks @matjazpotocnik I've pushed a fix for this issue. The issue actually was that the documentation wasn't correct. I was trying to communicate that if an item in `$a` was `label => url` then it would converted to the `<a>` tag, but I didn't communicate it correctly. I've attempted to add to the documentation for the correct way (it's a little hard to explain), but have also added support for the unintended option as well.
